### DISCUSSION TITLE
Import data csv - variant name x product name

### DISCUSSION THREAD

**Discussion by lucasrochaa** [vendurehq/vendure#1582] [Q&A]
Hello!

I'm stuckin in a scenario, i have one product and 3 variants.

product name: XXX
    
variant name 1 :  aaa
variant name 2 :  bbb
variant name 3 :  ccc
    
how i need to build the csv to achieve the scenario above.

Thanx!

**Comment by michaelbromley** [comment]
Hi!

Currently the CSV import format does not allow you to specify variant names - it simply concatenates the product name and any options.

If you need to specify variant names, you could write a custom import script, there are some pointers here: https://www.vendure.io/docs/developer-guide/importing-product-data/#custom-populate-scripts

**Reply by lucasrochaa** [reply]
Ok Michael, thanx for the tip!
